sw/device/silicon_creator/lib/BUILD

```diff
@@ -272,6 +273,8 @@ filegroup(
     ],
 )
 
+autogen_chip_info(name = "chip_info")
```

Comment:
1. `shutdown` is also used in `ROM_EXT`. We need to decide how many `chip_info`s we want. One for ROM and another one for ROM_EXT or just one in ROM? I think I prefer the latter since it includes the chip gen/rev fields. We can have something else for ROM_EXT if we need to.
2. If we want just one `chip_info` we must update `shutdown` so it always looks at the `chip_info` in ROM regardless of where it's printed from. If we want two `chip_info`s then do we want to make it clear where we are shutting down the chip, i.e. in ROM or ROM_EXT? Is it something we are OK with disclosing?

@moidx @cfrantz 

@dmcardle I don't think we have to resolve these right now. Please feel free to create an issue and we can discuss offline or in relevant WG meetings.

Reply:
IMHO, there should be one chip_info, and it should be in the ROM (ie: baked into the hardware).

Reply:
I agree. We should have it baked into hardware. 

wrt. disclosing, I always prefer to have the ability to redact information to be able to mitigate any potential attacks. Probably something to analyze with @bilgiday as part of the FI test plan.

Reply:
I've been thinking of this like we're tagging software artifacts with the intended hardware target. From that perspective, I think it makes sense to have a copy in ROM_EXT as well, since we could feasibly install the wrong ROM_EXT into flash.

Reply:
In the interest of unblocking this PR, I'd like to move this conversation to #18198. I'll strip the `chip_info` down to just the commit hash, which I think is unobjectionable, and I'll make sure that the ROM_EXT linker script drops `chip_info` for the time being.

Reply:
I've now scoped chip_info down to just the commit hash, and we've moved the conversation to #18198. Resolving this thread.